tests/cosim/elaborate/src/DUT.scala

```diff
@@ -0,0 +1,97 @@
+package cosim.elabotate
+
+import chisel3._
+import freechips.rocketchip.diplomacy.{AddressSet, BundleBridgeSource, InModuleBody, LazyModule, RegionType, SimpleLazyModule, TransferSizes}
+import freechips.rocketchip.interrupts.{IntSinkNode, IntSinkPortSimple, IntSourceNode, IntSourcePortSimple}
+import freechips.rocketchip.subsystem._
+import freechips.rocketchip.tilelink.{TLManagerNode, TLSlaveParameters, TLSlavePortParameters}
+import org.chipsalliance.cde.config.Parameters
+import freechips.rocketchip.tile.{NMI, PriorityMuxHartIdFromSeq, RocketTile}
+
+import org.chipsalliance.cde.config.{Config, Field}
+
+class DUT(p: Parameters) extends Module {
```

Comment:
You can directly use `BaseSubsystem` with `InstantiatesTiles` to construct a minimize version of LazyModule